conftest.py

```diff
@@ -40,6 +41,11 @@ def django_db_setup(django_db_setup, django_db_blocker, fixtures):
         call_command('loaddata', *fixtures)
         set_group_permissions()
 
+        # add the api user to the api group
```

Comment:
I dont understand why this needs to be here and why it does not come from the fixtures?
We have "Alex Api" in the fixtures..

Reply:
This was actually a bug, "Alex Api" is in the fixtures and in the database, but he is not in the `api` group. But you are right, this belongs in the fixture. I don't know why I thought this can only be added here ...

Reply:
I think that also the roles for "Manni Manager" are missing from the `accounts.json`.
He needs `[1,2,3]` right?
```json
{
    "model": "accounts.role",
    "pk": 6,
    "fields": {
      "user": 6,
      "member": [
        1
      ],
      "manager": [],
      "editor": [],
      "reviewer": []
    }
  },
```

Reply:
oh no, I forget we have the "Sandra Site" for this and should be give the manager roles for all of the sites.
And thanks for the correction @jochenklar, "Manni Manager" is the user for a specific project role only (like guest, owner, etc.). 
So the change for Manni should be reverted and Sander should get `[2,3]` in addition.